### ISSUE TITLE
Run the Godot games more faster

### ISSUE THREAD

**Issue by kaissouDev** [godotengine/godot#78874]
### Godot version

4.1.rc2

### System information

All

### Issue description

Hello, when I launch my game Godot there is a white screen that lasts between 1 to 2 seconds.

### Steps to reproduce

Juste run a Godot games 

### Minimal reproduction project

All Godot project

**Comment by AThousandShips** [comment]
Without any information about this we can't help, also for support questions and help please first turn to the other [community channels](https://godotengine.org/community), this is for bug reporting, not a help desk.

**Comment by kaissouDev** [comment]
Actually it's a performance problem when running Godot, but you're right I have to change Godot 4.0 to Godot 4.1.rc2

**Comment by AThousandShips** [comment]
Yes, but you asked it in the form of a support question, sorry for assuming based on that.

But this still isn't enough information

However I would assume you're describing:
* #74292

So this would be a duplicate, and I'd suggest searching for issues before opening

**Comment by kaissouDev** [comment]
Ok sorry for the inconvenience... You're right it's a duplicate issue of #74292

**Comment by AThousandShips** [comment]
Duplicate: #74292

**Comment by AThousandShips** [comment]
No worries! Pinned the issue to make it easier for people to catch in the future

**Comment by akien-mga** [comment]
Yes just to clarify, having to wait 1 or 2 s for the process to start is quite expected, Godot is a relatively complex application that needs to allocate a lot of memory, start a high end renderer, etc. There aren't a lot of applications nowadays that start in less than 1 s.

If we can identify ways to speed this up further, that would definitely be great, but it's not a bug.
